server/src-lib/Hasura/Server/Telemetry.hs

```diff
@@ -159,7 +159,7 @@ computeMetrics sc =
     calcPerms fn perms = length $ catMaybes $ map fn perms
 
     relsOfTbl :: TableInfo PGColumnInfo -> [RelInfo]
-    relsOfTbl = rights . Map.elems . Map.map fieldInfoToEither . _tiFieldInfoMap
+    relsOfTbl = getRels . _tiFieldInfoMap
```

Comment:
I think this function is short enough now that it’d be better to just inline it at its use-site, since it’s only used in one spot.